examples/terraform-aws-s3-example/main.tf

```diff
@@ -53,6 +53,16 @@ data "aws_iam_policy_document" "s3_bucket_policy" {
   }
 }
 
+resource "aws_s3_bucket" "test-bucket-logs" {
+  bucket = "${local.aws_account_id}-${var.tag_bucket_name}-logs"
+  acl    = "log-delivery-write"
+
+  tags = {
+    Name        = "${local.aws_account_id}-${var.tag_bucket_name}-logs"
+    Environment = var.tag_bucket_environment
+  }
+}
```

Comment:
Might want to set `force_destroy = true` to ensure it can be destroyed.

Reply:
good point!